docs/_build/advanced-features/bundle.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Reference user-entered details with data bundles
+order: 3
+layout: post-toc
+redirect_from: 
+---
+
+# Reference user-entered details with data bundles
+
+Zapier stores data from users' authentication and input forms for API calls in the `bundle` object. You can reference that data in your integration using {% raw %}`{{bundle.bundleName.field}}`{% endraw %} text in API requests and connection labels, replacing `bundleName` with the bundle name and `field` with the input field key or API response field key you need. You can also reference bundles in custom code if you switch to Code Mode for a request, using the same name but without the curly brackets, for example `return bundle.bundleName.field;` to have JavaScript code return a specific field.
+
+If an API response includes a nested field, you can reference it as `field.nestedfield`, for example {% raw %}`{{bundle.inputData.data.name}}`{% endraw %} to reference a `name` nested field inside the `data` field.
+
+Zapier integrations include the following bundles:
+
+- authData
+- inputData
+- rawRequest and cleanedRequest
+- targetURL
+- subscribeData
+- process.env
```

Comment:
good call @michellechu77 - I will add `meta`, missed it as it was never included in these docs but is in the CLI.
for `outputData` since it is only applicable in callback functions - do you know if those can be implemented in the UI? I've never seen that and wouldn't include it, if it requires the CLI to use, like here - https://github.com/zapier/zapier-platform/blob/main/packages/cli/README.md#bundleoutputdata